packages/android_intent/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.4.0
```

Comment:
nit: At this level pub would consider this a breaking change, should this be `0.3.7`? When plugins are pre 1.0.0 the schema is `0.major.minor+patch`.